package.json

```diff
@@ -13,14 +13,14 @@
     "LICENSE"
   ],
   "scripts": {
-    "lint": "# tslint \"src/**/*.ts\"",
-    "build": "rm -rf dist/ && tsc",
+    "lint": "echo deactivated for now: tslint src/**/*.ts",
+    "build": "rimraf dist && tsc",
     "dependency-check": "dependency-check . --entry dist/bin/typings-bundle.js --entry dist/bin/typings-init.js --entry dist/bin/typings-install.js --entry dist/bin/typings-list.js --entry dist/bin/typings-search.js --entry dist/bin/typings-uninstall.js --unused --no-dev -i bluebird",
     "test-spec": "ts-node node_modules/blue-tape/bin/blue-tape.js \"src/**/*.spec.ts\" | tap-spec",
     "test-cov": "ts-node node_modules/istanbul/lib/cli.js cover -e .ts --print none -x \"*.d.ts\" -x \"*.spec.ts\" blue-tape -- \"src/**/*.spec.ts\" | tap-spec",
     "test": "npm run lint && npm run build && npm run dependency-check && npm run test-cov",
-    "bootstrap": "mkdir -p typings && touch typings/main.d.ts && SKIP_PREPUBLISH=true npm install && npm run build 2> /dev/null; cat dist/init.js > /dev/null && node dist/bin/typings.js install",
-    "prepublish": "if [ ! $SKIP_PREPUBLISH ]; then npm run build; fi"
+    "bootstrap": "npm install shelljs && node scripts-bootstrap.js && node dist/bin/typings.js install",
```

Comment:
I'm installing `shelljs` by itself because `scripts-bootstrap.js` depends upon it.  However, I don't want to do a straight `npm install` because that would trigger the [prepublish script](https://docs.npmjs.com/misc/scripts):

> prepublish: Run BEFORE the package is published. (Also run on local npm install without any arguments.)

And I want to avoid that being triggered in this particular scenario.  I did experiment with setting an environment variable which would then be picked up in `prepublish` but that somehow was getting wiped so I switched to this approach.